packages/ng/forms/public-api.ts

```diff
@@ -10,6 +10,7 @@ export * from './switch-input/switch-input.component';
 export * from './radio-group-input/radio-group-input.component';
 export * from './radio-group-input/radio/radio.component';
 export * from './textarea-input/textarea-input.component';
+export * from './phone-number-input/index';
```

Comment:
We should move this component in its own lib to avoid adding libphonejs deps to all codebases. We could import it from `@lucca-front/ng/forms/phone-number-input`.

Reply:
I'm not a big fan of having such deep imports but you're right, it shouldn't be in all bundles if they're not using the phone number input